src/main/scala/rocket/ICache.scala

```diff
@@ -47,6 +47,7 @@ class ICache(val latency: Int, val hartid: Int)(implicit p: Parameters) extends
         regionType      = RegionType.UNCACHED,
         executable      = true,
         supportsPutFull = TransferSizes(1, wordBytes),
+        supportsPutPartial = TransferSizes(1, wordBytes),
```

Comment:
No, it's actually true.  Not adding it to the list of supported operations was an oversight.